### ISSUE TITLE
I would like to fetch all recent feed data from particular since time. 

### ISSUE THREAD

**Issue by Vignesh-balakrishnan14** [Podcastindex-org/docs-api#57]
When hitting the recent feed data API, I will get a maximum of 1000, if set the max as 1000. I want to get all recent feed data if the record has more than 1000. Please implement the pagination to retrieve all recent feed data.

**Comment by daveajones** [comment]
What is the use case for this?  What are you trying to do from your side?  There are a couple of other endpoints that may give you what you need.

**Comment by Vignesh-balakrishnan14** [comment]
Hi Dave,

Our case is we need to update the podcasts's latest episode details in our application. So, That we tried to use the recent feed API in PodcastIndex.org <http://podcastindex.org/> But we faced previous mentioned issues. Do you have suggestion?

Regards,

Vignesh

**Comment by daveajones** [comment]
So, you are trying to track every time any feed in the index is updated?

**Comment by Vignesh-balakrishnan14** [comment]
Yes, for every 1 or 2 hours manner.

**Comment by Vignesh-balakrishnan14** [comment]
Shall we have a call via google meet along with my one of the colleague?

**Comment by daveajones** [comment]
I'm working on a new update endpoint right now that will better suit your needs.  We're beta testing it currently.  I'll update this issue when it's ready.

**Comment by Vignesh-balakrishnan14** [comment]
Thanks @daveajones

**Comment by daveajones** [comment]
We're using object storage for tracking updates to the index now.  The most current feed/episode update data is here:

https://tracking.podcastindex.org/current

You can use the "previousTrackingUrl" of each data block to paginate back in time from wherever you are in the chain.  Details here:

https://podcastindex.social/@dave/106456341067494760

**Comment by Vignesh-balakrishnan14** [comment]
I checked this current feed/episode URL which is useful to us. But we want the iTunes podcast and English language podcasts alone. Have any possibility to add the iTunes ID in feeds array and language filtering in that current feed/episode URL?

**Comment by daveajones** [comment]
Yes I can add the iTunes id and language to the results. This isn’t a true API endpoint. It’s a data dump to object storage. So, you will need to filter out the languages on your side. This tracks every single change to the database.

The “recent” endpoints put a lot of load on our database, so doing it this way allows us to reduce that.

I will post here when the iTunesID and language are added shortly.

**Comment by Vignesh-balakrishnan14** [comment]
Okay. Thanks, @daveajones.

**Comment by Vignesh-balakrishnan14** [comment]
Do you have any idea to include the pagination on the recent feed API?

**Comment by daveajones** [comment]
Not at this time.

**Comment by Vignesh-balakrishnan14** [comment]
Okay.

**Comment by daveajones** [comment]
The itunes id is now added.  Language is already there.

**Comment by Vignesh-balakrishnan14** [comment]
Hi dave, We need the feed categories. Can I expect categories attribute in the current feed/episode data?

**Comment by daveajones** [comment]
Let me look and see...

**Comment by daveajones** [comment]
> One more clarification, current feed/episode URL data give currently updated episodes alone or newly created podcasts feed also?

It's both.  That is the full firehose of all updates, regardless if they are new or existing.

**Comment by Vignesh-balakrishnan14** [comment]
> Let me look and see...

Please consider. If you don't want to add that attribute. Please let me know.
